src/python/pants/backend/awslambda/python/target_types.py

```diff
@@ -107,27 +137,20 @@ class PythonAwsLambdaRuntime(PythonFaaSRuntimeField):
         """
     )
 
-    known_runtimes = (
-        PythonFaaSKnownRuntime(3, 6, LAMBDA_DOCKER_REPO, "3.6", FaaSArchitecture.X86_64),
-        PythonFaaSKnownRuntime(3, 7, LAMBDA_DOCKER_REPO, "3.7", FaaSArchitecture.X86_64),
```

Comment:
Hm, looks like we lost 3.6 and 3.7 support? Maybe an oversight? If not could you explain why 😄

Reply:
Not an oversight - AWS Lambda no longer supports these runtimes at all, so neither do we. [The lowest runtime they currently support is 3.8](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html).

Reply:
Python 3.6 is fully deprecated, based on the deprecation dates listed we could potentially bring back 3.7 since updates are allowed until Feb 28, 2025.

Reply:
Ah, I see, thanks for explaining. Can I suggest we do this as a separate change?

As it is, people might be using this for some real code and we'll be silently breaking them (e.g. maybe they've somehow convinced AWS to extend support by paying a whole lot, or are using the "lambda" targets for non-lambda deployments that they're compatible with).

If we go ahead with it, I think we should deprecate-then-remove over a few versions and call it out in release notes etc. However, given how little effort it is to support, I'm not feeling the need to urgently get rid of it. Happy to approve a PR that executes a smooth removal, though.

Reply:
Reverted this part of the change + added 3.6 and 3.7 back in.